src/Tribe/Commerce/PayPal/Oversell/Admin_Notice_Decorator.php

```diff
@@ -120,14 +170,31 @@ public function get_post_id() {
 	}
 
 	/**
-	 * Returns the policy nice name.
+	 * Returns the post edit link skipping the `current_user_can` check.
+	 *
+	 * This might happen in the context of a PayPal request handling and the
+	 * current user will be set to `0`.
+	 *
+	 * @param object $post_type
+	 * @param int    $post_id
+	 *
+	 * @return string
+	 */
+	protected function get_user_insensible_edit_link( $post_type, $post_id ) {
+		$action = '&amp;action=edit';
+
+		return admin_url( sprintf( $post_type->_edit_link . $action, $post_id ) );
```

Comment:
use `add_query_arg` to add the action, so we make sure it replaces if it already has it.